pom.xml

```diff
@@ -108,7 +108,7 @@
                         <configuration>
                             <referenceVersion>2.22ea3</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <!--                            <binaryCompatibilityPercentageRequired>89.4</binaryCompatibilityPercentageRequired>-->
+                            <binaryCompatibilityPercentageRequired>99.2</binaryCompatibilityPercentageRequired>
```

Comment:
EventLoops became a proper utility class (got a private constructor and was made final)